pisces/data_sets.py

```diff
@@ -745,12 +747,15 @@ def accelerometer_to_spectrogram(self, accelerometer: pl.DataFrame) -> np.ndarra
             )  # fill in the args given, which must be everything besides numerical input
 
             # apply
-            x_ = fn_args(x_)
-            y_ = fn_args(y_)
-            z_ = fn_args(z_)
+            for i in indexes_to_use:
+                data = acc[:, i]
+                processed_data.append(fn_args(data))
 
-        spec = x_ + y_ + z_
-        spec /= 3.0
+        spec = np.zeros_like(processed_data[0])
```

Comment:
instead of this, you can just do:
`spec = sum(processed_data) / len(processed_data)`